wpilibj/src/main/java/edu/wpi/first/wpilibj/command/Command.java

```diff
@@ -151,6 +198,25 @@ public Command(String name, double timeout) {
     m_timeout = timeout;
   }
 
+  /**
+   * Creates a new command with the given name and timeout.
+   *
+   * @param name    the name of the command
+   * @param timeout the time (in seconds) before this command "times out"
+   * @param requirement the subsystem that this command requires
+   * @throws IllegalArgumentException if given a negative timeout
+   * @throws IllegalArgumentException if given a negative timeout or name was null.
+   * @see Command#isTimedOut() isTimedOut()
+   */
+  public Command(String name, double timeout, Subsystem requirement) {
+    this(name);
+    if (timeout < 0) {
+      throw new IllegalArgumentException("Timeout must not be negative.  Given:" + timeout);
+    }
+    m_timeout = timeout;
```

Comment:
Wouldn't it make more sense to delegate to `this(name, timeout)` here? (Same for the others.)